crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -390,6 +390,41 @@ impl<'de> Deserialize<'de> for Entity {
     }
 }
 
+/// Outputs the full entity identifier, including the index, generation, and the raw bits.
+///
+/// This takes the format: `{index}v{generation}#{bits}`.
+///
+/// # Usage
+///
+/// Prefer to use this format for debugging and logging purposes. Because the output contains
+/// the raw bits, it is easy to check it against serialized scene data.
+///
+/// Example serialized scene data:
+/// ```text
+/// (
+///   ...
+///   entities: {
+///     4294967297: (  <--- Raw Bits
+///       components: {
+///         ...
+///       ),
+///   ...
```

Comment:
Super minor nit: closing parenthesis
```suggestion
///   ...
/// )
```